stock_account_valuation_report/models/product_product.py

```diff
@@ -48,21 +49,41 @@ def _search_valuation(self, operator, value):
 
     @api.model
     def _search_qty_discrepancy(self, operator, value):
-        products = self.env["product.product"].search([("type", "=", "product")])
-        pp_list = []
-        for pp in products:
-            if pp.qty_at_date != pp.account_qty_at_date:
-                pp_list.append(pp.id)
-        return [("id", "in", pp_list)]
+        products = self.env["product.product"].search(
```

Comment:
```suggestion
        products = self.with_context(active_test=False).env["product.product"].search(
```
and revert domain

Reply:
Thank you @MiquelRForgeFlow change done.

BTW what a horrible name "active_test" :disappointed: Nothing to do with a "test"